classes/security/authorization/OjsJournalMustPublishPolicy.inc.php

```diff
@@ -6,7 +6,7 @@
  * Copyright (c) 2000-2015 John Willinsky
  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
  *
- * @class OjsAuthorDashboardAccessPolicy
+ * @class OjsJournalMustPublishPolicy
```

Comment:
Also, the name of this class would be clearer if it was something like UnpublishedObjectAccessPolicy, I think...